lib/src/seq.rs

```diff
@@ -323,7 +375,7 @@ pub type PublicByteSeq = PublicSeq<u8>;
 
 /// Read hex string to Bytes.
 impl Seq<U8> {
-    #[cfg_attr(feature = "use_attributes", unsafe_hacspec)]
+    #[cfg_attr(feature = "use_attributes", not_hacspec)]
```

Comment:
This is actually used inside some hacspec primitives, for instance to read hex constants as an array of byte. Are you sure you want to disallow it?

Reply:
hm, really? I think this should only be used in tests.
But I really need to think about how to ensure in the typechecker that these functions aren't used.